tests/xlsx/xlsx_import.test.ts

```diff
@@ -721,18 +721,26 @@ describe("Import xlsx data", () => {
     [
       "line",
       [
-        { dataRange: "Sheet1!B26:B35", backgroundColor: "#7030A0" },
+        {
+          dataRange: "Sheet1!B26:B35",
+          backgroundColor: "#7030A0",
+          trend: { type: "polynomial", order: 2, display: true },
+        },
         { dataRange: "Sheet1!C26:C35", backgroundColor: "#C65911" },
       ],
     ],
     [
       "bar",
       [
-        { dataRange: "Sheet1!B27:B35", backgroundColor: "#7030A0" },
+        {
+          dataRange: "Sheet1!B27:B35",
+          backgroundColor: "#7030A0",
+          trend: { type: "exponential", display: true },
+        },
         { dataRange: "Sheet1!C27:C35", backgroundColor: "#C65911" },
       ],
     ],
-  ])("Can import charts %s with dataset colors", (chartType, chartDatasets) => {
```

Comment:
Can you make a separated test to check that we can only importe chart with trend line instead of modifying theses ?